topics/valkey-bundle.md

```diff
@@ -0,0 +1,173 @@
+
+# What is Valkey Bundle
+
+Valkey Bundle is a pre-packaged, containerized version of Valkey that comes preloaded with a collection of supported modules. These modules enable advanced data structures and search capabilities to extend Valkey’s core functionality. The bundle is designed to help developers quickly get started with powerful Valkey features without needing to manually install or configure anything. Some of the modules included in the bundle are:
+
+1. Valkey JSON - Allows users to natively store, query, and modify JSON data structures using the JSONPath query language.
+2. Valkey Bloom - Provides space-efficient probabilistic data structures, such as Bloom filters, for adding elements and checking whether they exist in a set.
+3. Valkey Search - Enables the creation of indexes and similarity searches through the use of full-text and complex filters. 
+4. Valkey LDAP - Handles user authentication against LDAP based identity providers.  
+
+# Quick Start to Using the Bundle
+
+The fastest way to start using Valkey Bundle is by downloading the official image through [Docker Hub](https://hub.docker.com/r/valkey/valkey-extensions/). The following steps will guide you through launching and interacting with your first instance!
+
+1. **Pull the image to get the latest public release**
+
+    ```bash
+    docker pull valkey/valkey-extensions
+    ```
+
+    This command downloads the most recent stable image of the Valkey Bundle, which includes the Valkey server along with the preloaded modules.
+
+    The Valkey Bundle image also supports multiple tags, allowing you to control the specific version and operating system base. This allows for more control over the environment, whether you’re aiming for a reproducible build (using a version like 8.1-bookworm) or a minimal footprint (alpine variant).
+
+    Check out the [Valkey Bundle Docker Hub Tag](https://hub.docker.com/r/valkey/valkey-extensions/tags) section to view all available tags and example pull commands. 
+
+2. **Start the Valkey server inside a container**
+
+    ```bash
+    docker run --name my-valkey-extensions -d -p 6379:6379 valkey/valkey-extensions
+    ```
+
+    This command starts the Valkey server in a background container named my-valkey-extensions and maps Valkey’s default port 6379 to your local machine for external access. By default, it uses the latest available image. To run a specific version or variant, append the desired tag to the image name. For example:
+
+    ```bash
+    docker run --name my-valkey-extensions -d -p 6379:6379 valkey/valkey-extensions:8.1-bookworm
+    ```
+ 
+3. **Connect to the server using valkey-cli**
+    
+    To interact with the server, use the Valkey command-line interface (valkey-cli). The example below runs the CLI from a separate container on the same Docker network.
+     
+    ```bash 
+    docker run -it --network some-network --rm valkey/valkey-extensions valkey-cli -h my-valkey-extensions
+    ```
```

Comment:
I'm not really sure why you would do this. I think someone should either have the cli installed on their local machine or they should run the cli from inside the environment. There is really no good reason to run it from another container. I would remove this specifically because it won't work as written since there is no "some-network".